soh/soh/SohMenuBar.cpp

```diff
@@ -890,6 +890,32 @@ void DrawEnhancementsMenu() {
 
                 UIWidgets::Spacer(0);
 
+                if (ImGui::BeginMenu("Frogs Ocarina Game")) {
+                    UIWidgets::EnhancementCheckbox("Customize Behavior", CVAR_ENHANCEMENT("CustomizeFrogsOcarinaGame"));
+                    UIWidgets::Tooltip("Turn on/off changes to the frogs ocarina game behavior");
+                    bool disabled = !CVarGetInteger(CVAR_ENHANCEMENT("CustomizeFrogsOcarinaGame"), 0);
+                    static const char* disabledTooltip =
+                        "This option is disabled because \"Customize Behavior\" is turned off";
+                    UIWidgets::PaddedEnhancementCheckbox("Instant Win", CVAR_ENHANCEMENT("InstantFrogsGameWin"), true, false, disabled, disabledTooltip);
+                    UIWidgets::Tooltip("Skips the frogs ocarina game");
+                    UIWidgets::PaddedEnhancementCheckbox("Unlimited Playback Time", CVAR_ENHANCEMENT("FrogsUnlimitedFailTime"), true, false, disabled, disabledTooltip);
+                    UIWidgets::Tooltip("Removes the timer to play back the song");
+                    bool disabledFrog = 0;
+                    static const char* disabledFrogTooltip =
+                        "This option is disabled because \"Customize Behavior\" is turned off or \"Unlimited Playback Time\" is on";
+                    if (!CVarGetInteger(CVAR_ENHANCEMENT("CustomizeFrogsOcarinaGame"), 0) == 0 || CVarGetInteger(CVAR_ENHANCEMENT("FrogsUnlimitedFailTime"), 0)) {
```

Comment:
```suggestion
                    if (!CVarGetInteger(CVAR_ENHANCEMENT("CustomizeFrogsOcarinaGame"), 0) || CVarGetInteger(CVAR_ENHANCEMENT("FrogsUnlimitedFailTime"), 0)) {
```
Just missed this bit here I think